website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -108,20 +109,55 @@ To delete a published tag to re-tag, follow these steps:
     git push -d origin vx.y.z
 ```
 
-## Deploying ORCA buckets, RDS cluster and Cumulus ORCA modules in bamboo
+## Deploying ORCA buckets, RDS cluster and Cumulus ORCA modules via bamboo
 
-For testing purposes, you should use your feature branch in cumulus-orca github repo and `prototype-latest` bamboo plan so that it does not affect the ORCA github `develop` branch. The `prototype-latest` bamboo plan is linked to `feature/ORCA-test-bamboo` github branch as the default branch under cumulus-orca repo. In addition, if changes are made to the bamboo spec file `bamboo.yaml` in this default branch, you have to manually import the bamboo spec by choosing `ORCA-test-branch` as the linked repo.
+For testing purposes, you should use your feature branch in cumulus-orca github repo and [`prototype-latest`](https://ci.earthdata.nasa.gov/browse/ORCA-PL) bamboo plan so that it does not affect the ORCA github `develop` branch. The `prototype-latest` bamboo plan is linked to `feature/ORCA-test-bamboo` github branch as the default branch under cumulus-orca repo. In addition, if changes are made to the bamboo spec file `bamboo.yaml` in this default branch, you have to manually import the bamboo spec by choosing `orca test branch` as the linked repo.
 
-Make sure to use the `ORCA Deploy Plan` bamboo plan for deploying the resources. These are the ORCA buckets that are created in disaster recovery AWS account:
+:::warning
+You should reset `feature/ORCA-test-bamboo` before using it.
+1. Rename `feature/ORCA-test-bamboo` to `feature/ORCA-test-bamboo-old`
+1. Create a new branch based off of your branch named `feature/ORCA-test-bamboo`

```

Comment:
not sure why we would want to do these steps

Reply:
Unless it is regularly synced with `develop` (not via manual, prone-to-error updates) it is invalid as a testing environment.

Reply:
there will be few parameters that will have to be different in both branches like the `repositories:` and `key`.

We can rebase the feature/ORCA-test-bamboo with develop branch and then do the testing.

Reply:
It was already based off of develop the first time, correct? In that case, rebase wouldn't solve anything. You'd still have the merge conflicts.

Reply:
yes based off develop.

If the above steps worked for you, then I am fine with adding them